tests/RightSignature/DocumentTest.php

```diff
@@ -109,4 +109,46 @@ public function testDocumentDetails()
         $this->assertEquals('support@rightsignature.com', $document->recipients[0]->email);
         $this->assertEquals('Document emailed to John Bellingham (john.b@rightsignature.com)', $document->audit_trail[0]->message);
     }
+
+    public function testSendDocument()
+    {
+        $response = <<<EOS
+            <document>
+                <status>sent</status>
+                <guid>2VMW88J3424MPEYF9DU6VY</guid>
+            </document>
+EOS;
+        $payload = [
+            'subject' => '- email subject -',
+            'action' => 'send',
+            'type' => 'base64',
+            'recipients' => [
+                'recipient' => [
+                    [
+                        'is_sender' => true,
+                        'role' => 'cc',
+                    ],
+                    [
+                        'name' => 'Signer 1',
+                        'email' => 'pjafwcyv@sharklasers.com',
+                        'role' => 'signer',
+                    ],
+                ],
+            ],
+        ];
+
+        $client = \Mockery::mock('client');
+        $client->shouldReceive('post')
+                ->withAnyArgs()
+                ->andReturn($response);
+
+        $file = fopen('sample-file.pdf', 'w');
```

Comment:
Perhaps you could use http://php.net/manual/en/function.tmpfile.php to create the input document. That way the file doesn't hang around the repo if the test crashes before `unlink` gets called.